src/com/facebook/buck/features/project/intellij/IjProjectTemplateDataPreparer.java

```diff
@@ -493,6 +493,8 @@ private void addAndroidResourcePaths(
     if (resourcePaths.isEmpty()) {
       androidProperties.put(RESOURCES_RELATIVE_PATH_TEMPLATE_PARAMETER, EMPTY_STRING);
     } else {
+
```

Comment:
nit: unnecessary changes here?

Reply:
god catch. I think I ran google java format on this and didnt need the changes after. ill remove